ev3sources/lms2012/c_com/source/c_com.c

```diff
@@ -2252,23 +2252,20 @@ void      cComSystemCommand(RXBUF *pRxBuf, TXBUF *pTxBuf)
       pReplyListHandles->Cmd      =  LIST_OPEN_HANDLES;
       pReplyListHandles->Status   =  SUCCESS;
 
-      for (HCnt1 = 0; HCnt1 < ((MAX_FILE_HANDLES/8) + 1); HCnt1++)
+      for (NByte = 0; NByte < (MAX_FILE_HANDLES / 8 + 1); NByte++)

```

Comment:
If `MAX_FILE_HANDLES / 8 + 1` is trying to calculate the number of bytes needed to store `MAX_FILE_HANDLES` bits, it could be better to use `(MAX_FILE_HANDLES + 7) / 8`.